fetch-downloads.py

```diff
@@ -0,0 +1,132 @@
+"""
```

Comment:
if occurs to me that this should probably be written in ruby, given the rest of the site is. Would cut down the docker size slightly, anyway :p

Reply:
Funny enough, the docker size doesn't change. This is because we use a multi-stage Dockerfile. So earlier stages are dismissed in the final result :D

But yes, if I could write Ruby, I would have :P